src/routes/blog/posts/v3-1/+page.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Announcing Files v3.1
+description: System tray icon, open minimized on startup, Smart Extract, prioritize files when sorting, and a new drag & drop gesture.
+thumbnail: /blog-resources/v3-1/Hero.jpg
+date: 1/9/2024
+author: files-community
+---
+
+We're excited to announce that starting today, v3.1 is now available. If you already have Files, there should be a notification on the toolbar to install the update. Otherwise, you can download Files for free from our [download page](/download/). You can also purchase Files on the [Microsoft Store](ms-windows-store://pdp/?ProductId=9nghp3dx8hdx&cid=FilesWebsite) or sponsor us on [GitHub](https://github.com/sponsors/yaira2) to help support the project.
+
+**TL;DR:** Files v3.1 adds an icon to the system tray, support for opening a minimized window on startup, smart extract, an option to prioritize files when sorting, group files by day, and more. Continue reading to learn more about these changes.
+
+## What's new in v3.1
+
+### System tray icon
+
+With the recently added option to keep Files open in the background, a popular request has been to add an icon to the system tray. Besides for indicating when there is an open process, right clicking the icon provides a shortcut to the documentation site and a way to fully close the background process.
+
+<figure>
+    <img src="/blog-resources/v3-1/SystemTray.jpg" alt="System tray icon" />
+</figure>
+
+### Open minimized on Windows Startup
+
+Enabling the option to "Open Files on Windows Startup" and the option to "Keep Files running in the background" will now open Files in the background when starting Windows.
+
+### Extract archives with "Smart Extract"
+
+Smart Extract is a new feature that will automatically choose the extraction mode based on the archive contents. If the archive contains a single file or folder, Smart Extract will extract the archive to the current directory, otherwise if there are multiple items, they will be extracted to a new folder. Smart Extract can be accessed via the `ctrl` + `shift` + `e` shortcut as well as the right click context menu.
+
+<figure>
+    <img src="/blog-resources/v3-1/SmartExtract.jpg" alt="Smart Extract" />
+</figure>
+
+### Prioritize files when sorting
+
+We added an option to prioritize files when sorting. Enabling this option will sort files and folders separately with files being placed at the top of the list. This feature is an addition to the existing options to "sort folders first" and "sort files and folders together". 
+
+<figure>
+    <img src="/blog-resources/v3-1/SortFilesFirst.jpg" alt="Prioritize files when sorting" />
```

Comment:
In the future, we should probably avoid JPGs for text due to their compression algorithm being prone to create artifacts on text.

Reply:
Which format do you recommend?

Reply:
PNG is usually a good choice since it has lossless compression. Webp is also a good choice since it can compress even more than PNG while being lossless (or alternatively allowing lossy compression for even more size reduction).

Reply:
I'll update them 👍